### PR TITLE
Documentation for container component

### LINKED ISSUE #3360: Documentation for container component
Add documentation for container component to https://docs.luigi-project.io. once the feature is finalized.

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Documentation for container component 

**Related issue(s)**
Resolves #3360

### CODE REVIEW

**Review by JohannesDoberer — APPROVED** [review]
LGTM

File: docs/getting-started.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Luigi consists of two main parts:
 * [General settings](general-settings.md) - other settings that can be configured in Luigi, such as HTML attributes, third-party cookie configuration and more.
 * [API](luigi-core-api.md) - functions to help with almost every part of your app: navigation, authorization, adding a [search box](luigi-core-api.md#globalsearch), configuring a light/dark [theme](luigi-core-api.md#theming) and others.
 
+<!-- add-attribute:class:success -->
+>**NOTE:** As of Luigi v2.5, you can use [Luigi Container](luigi-container.md) to embed a Luigi micro frontend into any existing application without the need of it being a Luigi Core app.
```
Comment (hardl) [inline docs/getting-started.md:58]: Luigi container is detached from core/client versioning and follows it's own release cycle

File: docs/luigi-container.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```html
+    <luigi-compound-container 
+        context='{"label": "Dashboard"}'
+        compoundConfig = { your config here }>
+    </luigi-compound-container>
+```
+
+## Example
+
+1. You can find a Luigi Container example on [GitHub](https://github.com/SAP/luigi/tree/main/container). First, first clone the Luigi repository if you haven't already done so:
```
Comment (hardl) [inline docs/luigi-container.md:65]: redundant "first" ;)

**Review by hardl — CHANGES_REQUESTED** [review]

**Review by hardl — APPROVED** [review]
LGTM!

**Pull request merged**